Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>target/\ndependency-reduced-pom.xml\n\n### STS ###\n.apt_generated\n.classpath\n.factorypath\n.project\n.settings\n.springBeans\n.sts4-cache\n\n### IntelliJ IDEA ###\n.idea\n*.iws\n*.iml\n*.ipr
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 534f6c01cd95863674afbbf115482de008c9ec1d)
+++ b/.gitignore	(date 1711946478391)
@@ -14,4 +14,6 @@
 .idea
 *.iws
 *.iml
-*.ipr
\ No newline at end of file
+*.ipr
+
+spring-shell.log
\ No newline at end of file
